portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,64 +34,62 @@
 /** Returns an array of comments in JSON form */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  int maxComments=3;
+  private int numComments = 3;
   DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  /* Returns an array of comments in JSON 
-   * Size of array is maxComments */
+
+  /** Returns an array of comments in JSON of size numComments */
   @Override
   public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
-    PreparedQuery results = datastore.prepare(query);
-    ArrayList<String[]> comments = new ArrayList<String[]>();
-    int index=0;
-    for (Entity entity : results.asIterable()) {
-      if(index<maxComments){
-        String[] str ={(String) entity.getProperty("username"),(String) entity.getProperty("text")};
-        comments.add(str);
-      }
-      index++;
-    }
+    ArrayList<String[]> comments = getComments();
```

Comment:
Not necessarily for this PR, but it could be helpful to create a Comment class instead of just turning it into a String